<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RestoreWindow</class>
 <widget class="QWidget" name="RestoreWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Restore</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Select backup to restore:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="backupList"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Restore to:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="restorePathEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="browseButton">
       <property name="text">
        <string>Browse...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="restoreButton">
     <property name="text">
      <string>Start Restore</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui> 